plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1668,7 +1668,7 @@ public void setEnvVariables(String[] env) {
                     Log.log(e);
                 }
             } else {
-                Log.logInfo("Expected: " + new File(condaPrefix, "/etc/conda/activate.d")
+                Log.logInfo("Expected: " + new File(condaPrefix, "/")
```

Comment:
I think this could be just `condaPrefix` instead of `new File(condaPrefix, "/")`.